lib/src/open_prices_api_client.dart

```diff
@@ -1,35 +1,81 @@
 import 'package:http/http.dart';
+import 'package:http_parser/http_parser.dart';
+import 'package:http/http.dart' as http;
+import 'package:path/path.dart';
 
+import 'prices/maybe_error.dart';
+import 'prices/price.dart';
+import 'prices/proof.dart';
+import 'prices/currency.dart';
 import 'prices/get_prices_result.dart';
 import 'prices/get_prices_results.dart';
 import 'prices/location.dart';
+import 'prices/location_osm_type.dart';
 import 'prices/price_product.dart';
+import 'prices/proof_type.dart';
+import 'prices/session.dart';
 import 'prices/validation_errors.dart';
 import 'utils/http_helper.dart';
 import 'utils/open_food_api_configuration.dart';
 import 'utils/uri_helper.dart';
+import 'utils/uri_reader.dart';
 
 /// Client calls of the Open Prices API.
+///
+/// cf. https://prices.openfoodfacts.org/api/docs
 class OpenPricesAPIClient {
   OpenPricesAPIClient._();
 
-  /// Subdomain of the Elastic Search API.
+  /// Status when the server is running (cf. [getStatus]).
+  static const String statusRunning = 'running';
```

Comment:
@M123-dev Not sure it would be appropriate: basically we just want to know if the server is "running" (I even hesitated returning a `bool` instead).
Using an `enum` would be problematic: what if there's a new status we don't know? We should then maintain the values. For no added values. Let's be resilient.